modern/org/commcare/modern/database/TableBuilder.java

```diff
@@ -180,7 +180,7 @@ public String getTableCreateString() {
 
     //sqlite doesn't like dashes
     public static String scrubName(String input) {
-        return input.replace("-", "_");
+        return input.replace("-", "_dash_").replace(".", "_dot_").replace("+", "_plus_");
```

Comment:
Oh, I guess if you have verified that no current scrubbed columns etc use `-`, `.`, or `+`, then we should be good.